tmva/tmva/src/MethodDL.cxx

```diff
@@ -443,14 +451,25 @@ void MethodDL::ParseInputLayout()
    int subDim = 1;
    std::vector<size_t> inputShape;
    inputShape.reserve(inputLayoutString.Length()/2 + 2);
-   inputShape.push_back(30);    // Will be set by Trainingsettings, use default now
+   inputShape.push_back(30);    // Will be set later by Trainingsettings, use 0 value now
```

Comment:
Why do you push back `30` here? I don't understand it, also not the comment after the line.